holoviews/plotting/mpl/tabular.py

```diff
@@ -89,7 +89,7 @@ def pprint_value(self, value):
         elif isinstance(value, str):
             formatted = safe_unicode(value)
         else:
-            formatted = str(value)
```

Comment:
Right, this change makes the `` elif isinstance(value, str):`` condition pointless. What would you recommend? Simply using ``str()`` results in unicode errors.